xpsi/HotRegion.py

```diff
@@ -265,8 +265,6 @@ def __init__(self,
         The phase of the hot region, a periodic parameter [cycles].
         """
         phase_bounds = bounds.get('phase_shift', None)
-        if not phase_bounds or None in phase_bounds:
```

Comment:
have you tried executing code with this change?

I do not believe this fixes the problem because `phase_bounds` is assumed to be an iterable in the next check.

Reply:
Right, that also needs to be tended to. I am yet to re-run the ST-S model with changes to HotRegion.py. Having made the changes listed in this pull-request, I had re-run the ST-U model which worked but that didn't rely on this bug.